advanced_functionality/tensorflow_iris_byom/tensorflow_BYOM_iris.ipynb

```diff
@@ -22,6 +22,13 @@
     "__Note__: Compare this with the [tensorflow bring your own model example](../tensorflow_iris_byom/tensorflow_BYOM_iris.ipynb)"
```

Comment:
<p>What is the reason for switching from a separate .py file to having the code in the notebook? Just being more cohesive, or anything else? Since it's only ~30 lines of code, I don't mind having it directly in the notebook like this.</p><p> </p><p>Related, why is an <code>entry_point</code> no longer needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Because the model is so small, I think a lot of folks would just define it in a notebook cell, especially since we need to load it for on-notebook training anyhow. If it were larger, or required additional processing tasks, I'd probably put it in it's own file.

As for the entry_point, as I understand it when you run `.save` on a Tensorflow model it includes all the code necessary to run it in the `saved_model.pb` file.

https://www.tensorflow.org/guide/saved_model

So, the TensorFlow estimator must have some logic to extract that automagically and use it as the inference fn. ¯\_(ツ)_/¯